src/client.cc

```diff
@@ -47,13 +47,14 @@ Client::Client(std::unique_ptr<UserInterface>&& ui,
     m_ui->set_ui_options(m_window->options()["ui_options"].get<UserInterface::Options>());
     m_ui->set_on_key([this](Key key) {
         kak_assert(key != Key::Invalid);
-        if (key == ctrl('c'))
+        auto opts = context().options();
+        if (key == opts["interrupt_key"].get<Key>())
```

Comment:
so this means that if a user really wants to, they can map it to different
keys in different modes with `ModeChange` hooks.
I wonder if we want to make this more ergonomic, I don't have a strong
opionion on this, this seems fine.
I guess `<c-g>` is only ever useful in normal or prompt mode, so it likely doesn't matter.